components/Transitions/Appear.js

```diff
@@ -4,38 +4,40 @@ import { Transition } from 'react-transition-group';
 import anime from 'animejs';
 import { getCSSVariableAsNumber, getCSSVariableAsObject } from '../../utils/CSSVariables';
 
-const defaultEasing = getCSSVariableAsObject('--animation-easing-js');
+const AppearTransition = ({ children, ...props }) => {
+  const defaultEasing = getCSSVariableAsObject('--animation-easing-js');
 
-const appear = () => ({
-  opacity: [0, 1],
-  scale: [0, 1],
-  elasticity: 0,
-  easing: defaultEasing,
-  duration: getCSSVariableAsNumber('--animation-duration-standard-ms'),
-});
+  const appear = () => ({
```

Comment:
Are the appear/disappear functions redundant when doing it this way?

Either:

```js
const AppearTransition = ({ children, ...props }) => {
  const defaultEasing = getCSSVariableAsObject('--animation-easing-js');

  const appear = {
    easing: defaultEasing,
    duration: getCSSVariableAsNumber('--animation-duration-standard-ms'),
    ...
  };

  return (
    <Transition
      onEntering={el => anime({ targets: el, ...appear })}
      {...props}
    >
      { children }
    </Transition>
  );
};
```

Or in previous style:

```js
const defaultEasing = () =>
  getCSSVariableAsObject('--animation-easing-js');

const appear = () => ({
  opacity: [0, 1],
  scale: [0, 1],
  elasticity: 0,
  easing: defaultEasing(),
  duration: getCSSVariableAsNumber('--animation-duration-standard-ms'),
});

const AppearTransition = ({ children, ...props }) => (
  <Transition
    onEntering={el => anime({ targets: el, ...appear() })}
    {...props}
  >
    { children }
  </Transition>
);
```

Reply:
While both of those would also have fixed this, its honestly more time to write out the reply than to resolve the issue, so *shrug*.

Reply:
That was not meant as snarky btw...genuinely do go ahead and change it if you'd like!